### ISSUE TITLE
Passing flags to the ruby comment inside Gemfile

### ISSUE THREAD

**Issue by iainbeeston** [rvm/rvm#1823]
TL;DR If you specify any flags for your ruby version in your Gemfile (using the `#ruby= comment`) rvm replaces '--' with '-'

I'm using a comment in my Gemfile to specify my ruby version, eg.

``` ruby
#ruby=rbx-2.0.0.rc1
```

However, I'm using rubinius 2 I'd really like to make sure everyone on my project is using rubinius in ruby 1.9 mode and by default rubinius 2 compiles in ruby 1.8 mode. To get around that, I've specified the following instead of the above:

``` ruby
#ruby=rbx-2.0.0.rc1 --1.9
```

But when I navigate into a newly checked out copy of my project, rvm shows this message:

```
rbx-2.0.0.rc1 -1.9 is not installed.
To install do: 'rvm install rbx-2.0.0.rc1 -1.9'
```

which is an invalid command because there should be two dashes, not one (running it returns "Unrecognized command line flag: '-1.9' ( see: 'rvm usage' )"), as there is in the Gemfile.

I'm using the latest version of rvm (stable).
